vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -138,7 +138,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>winsqlite3.lib;ChakraRT.lib;dxguid.lib;dloadhelper.lib;OneCoreUap_apiset.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <DelayLoadDLLs>chakra.dll;winsqlite3.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
+      <DelayLoadDLLs>api-ms-win-core-file-l1-2-0.dll;ext-ms-win-uiacore-l1-1-1.dll;api-ms-win-core-windowserrorreporting-l1-1-0.dll;chakra.dll;winsqlite3.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
```

Comment:
>api [](http://example.com/codeflow?start=21&length=3)

The description mentions uiautomationcore.dll but it does not appear to be present on the delayload list.  Is that intentional?

Reply:
IOW UIAutomationCore is the implementation DLL for `ext-ms-win-uiacore-l1-1-1.dll`

Reply:
Ah, that makes sense.  I didn't look far enough past the `ext-` prefix.  (Also you don't depend on them by name but you do depend on them via onecoreuap_apiset.lib which includes a whole bucket of api- and ext- libraries).